src/main/java/com/KacFlor/ShopSpring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,37 @@
+package com.KacFlor.ShopSpring.controller;
+
+import com.KacFlor.ShopSpring.security.AuthenticationRequest;
+import com.KacFlor.ShopSpring.security.AuthenticationResponse;
+import com.KacFlor.ShopSpring.service.AuthenticationService;
+import com.KacFlor.ShopSpring.security.RegisterRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
I would change this url to "authentication"

Reply:
done